GOVERNANCE.md

```diff
@@ -0,0 +1,195 @@
+
+- Push access to some of the repositories of the [Biome GitHub org][gh-org]
+- `Maintainer` status on the [Biome Discord server][discord]
+- Ability to [vote](#voting) on project decisions
+- Ability to [vote](#maintainer-nomination) on new maintainers
+
+#### Maintainer nomination
+
+Maintainers may either be nominated by another [maintainer](#maintainer), [Core contributor](#core-contributor), [lead](#lead).
+
+When nominating a new maintainer, the person has to create a new thread in the `#maintainers` channel, provide a link to the history of the person's contributions to the project and a brief explanation of why the person should join as [maintainer](#maintainer).
+
+`#maintainers` is a private channel available only to people who have the `maintainer` role.
+
+The voting period will follow the [voting rules](#voting-rules). An individual becomes a [maintainer](#maintainer) if:
+- reaches more than 50% of positive votes among the people who have voting rights;
+- if the voting session ends with 50%, the Lead maintainers will take the decision;
+
+A maintainer can be self-nominated. They will have to message a [lead maintainer](#lead) privately with links to their contributions to the project. Then, the lead can:
+- reject the nomination in place, provide reasons why and possibly some suggestions for the future;
+- initiate a voting session among the maintainers;
+
+Upon termination of a nomination, the thread will be deleted, and a [core contributor](#core-contributor) will reach out to the nominee and ask them if they want to accept the title.
+
+In the event of a rejection, the nominated person will be privately given the requirements they have not met. Details of the discussion, such as the names of those who objected the nomination, will not be disclosed.
+
+
+### Voting Rules
+
+- Members may abstain from a vote.
+- Members who do not vote within seven days will automatically abstain.
+- Leads may reduce the seven days automatic abstain for urgent decisions.
+- Leads reserve the right to veto approval with a publicly disclosed reason.
+
+### Ownership
+
+One or more people can own certain parts of the codebase. This process is informal, and inclusion could result from substantial contributions or delegation by other members. A maintainer's responsibility is to identify the relevant owners and ensure there's an understanding when it comes to code review.
+
+
+### Inactivity
+
+There are no expectations around activity once someone becomes a core contributor or maintainer. Inactive core contributors or maintainers may have voting rights removed; however, will always retain their status. A core contributor or maintainer may request their voting rights back upon sufficient activity.
+
+
+## Current Members
+
+Members are listed in alphabetical order. Members are free to use the full name, GitHub handle, or any other nickname they wish to be addressed. Members are free to disclose their gender.
```

Comment:
As a non-binary person, I appreciate the last part here, but generally disclosing gender isn't something I'd expect to be done. My _guess_ is that you meant pronouns and used gender as a proxy for that, so here's two suggestions on how to tweak this:

```suggestion
Members are listed in alphabetical order. Members are free to use the full name, GitHub handle, or any other nickname they wish to be addressed. Members are free to disclose their gender and pronouns.
```

```suggestion
Members are listed in alphabetical order. Members are free to use the full name, GitHub handle, or any other nickname they wish to be addressed. Members are free to disclose their pronouns.
```